src/diamond/collector.py

```diff
@@ -254,6 +257,8 @@ def get_default_config_help(self):
                                  'Mutually exclusive with metrics_blacklist',
             'metrics_blacklist': 'Regex to match metrics to block. ' +
                                  'Mutually exclusive with metrics_whitelist',
+            'tags': '(optional) tags in format [[[tags]]]'
+            + ' to be attached to the metrics',
```

Comment:
This message is still wrong.
It says that in config I can write
tags: [[[some tags]]]